app/share/utils.py

```diff
@@ -0,0 +1,30 @@
+from tempfile import NamedTemporaryFile
+from flask import current_app
+from werkzeug.utils import secure_filename
+import os
+from app import s3
+import subprocess
+import uuid
+
+
+def handle_upload(file_field):
+    path = upload(file_field.data)
+    return path
+
+
+def upload(image_pc):
```

Comment:
While uploading image, you might need to create a filter for supported image files.
The unsupported files might need to throw a "unsupported file error" message.
At the moment, even a pdf gets uploaded as the story